fm.reaper.Reaper.yml

```diff
@@ -0,0 +1,83 @@
+app-id: fm.reaper.Reaper
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+tags:
+  - proprietary
+command: reaper
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0
+  - --device=all
+  - --filesystem=home
+
+add-extensions:
+  org.freedesktop.LinuxAudio.Plugins:
+    directory: extensions/Plugins
+    version: "22.08"
+    add-ld-path: lib
+    merge-dirs: lv2;vst;vst3
+    subdirectories: true
+    no-autodownload: true
+
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: "22.08"
+    add-ld-path: "."
+    autodelete: false
+
+modules:
+  - name: reaper
+    buildsystem: simple
+    build-commands:
+      # Install script to extract tarball
+      - install -D apply_extra /app/bin/apply_extra
+      # Install wrapper script to run application
+      - install -D reaper /app/bin/reaper
+      # Add desktop integration
+      - install -Dm644 fm.reaper.Reaper.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      # Install flatpak metadata
+      - install -Dm644 fm.reaper.Reaper.metainfo.xml /app/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      # extensions
+      - install -d /app/extensions/Plugins
+      - install -d /app/lib/ffmpeg
+    sources:
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          # Extract downnloaded tarball
+          - mkdir -p export/share
+          - tar xf reaper.tar.xz -C /app/extra/export/share
+          - rm -f reaper.tar.xz
+          # Run installer script
+          - cd export/share/reaper_linux_*
+          - ./install-reaper.sh --quiet --install /app/extra
+          # Save icon
+          - install -Dm644 /app/extra/REAPER/Resources/main.png $FLATPAK_APP_DIR/share/icons/hicolor/${size}x${size}/apps/${FLATPAK_ID}.png
```

Comment:
as I said, the icon needs to be installed by the package, not the extra data. because at build time the extra data doesn't exists. Not sure about which icon to use though.

Reply:
Thanks. Now it's working nicely.